yaml/bedaisy.yaml

```diff
@@ -0,0 +1,35 @@
+Name: bedaisy.sys
+Author: Wack0
+Created: '2023-04-22'
+MitreID: T1068
+Category: vulnerable driver
+Verified: 'TRUE'
+Commands:
+  Command: sc.exe create BEDaisy.sys binPath=C:\windows\temp\BEDaisy.sys type=kernel
+    && sc.exe start BEDaisy.sys
+  Description: 'BattlEye Anti-Cheat BEDAISY.SYS PPL privesc.'
+  Usecase: Elevate privileges
+  Privileges: kernel
+  OperatingSystem: Windows 10
+Resources:
+- https://github.com/magicsword-io/LOLDrivers/issues/23
+- https://infosec.exchange/@Rairii/109310279380973806
+Acknowledgement:
+  Person: 'Wack0'
+  Handle: 'Wack0'
+Detection: []
+KnownVulnerableSamples:
+- Filename: ''
```

Comment:
```suggestion
- Filename: 'BEDAISY.SYS'
```

Reply:
We need this since the image name of a driver can be anything so we can't rely on it when extracting the enrichments. Hence why if know the "legitimate" name, I think we should add it.